src/Components/Login/Login.tsx

```diff
@@ -30,12 +30,14 @@ export default function Login({saveAdminData}) {
       localStorage.setItem('userToken', response.data.token);
       saveAdminData()
       navigate("/dashboard")
-      toast.success("Successfully")
+      toast.success(response?.data?.message ||"Successfully")
     })
     .catch((error)=> {
       // console.log(error?.response?.data?.message);
-      toast.error(error?.response?.data?.message)
-      setIsLoding(false)
+      toast.error(error?.response?.data?.message || "Invaild Data")
```

Comment:
Using the message "Invalid data" might be misleading to users and that they entered something wrong, however, it might be any other error other than that.